bundle/phases/deploy.go

```diff
@@ -14,10 +17,90 @@ import (
 	"github.com/databricks/cli/bundle/permissions"
 	"github.com/databricks/cli/bundle/python"
 	"github.com/databricks/cli/bundle/scripts"
+	"github.com/databricks/cli/libs/cmdio"
+	terraformlib "github.com/databricks/cli/libs/terraform"
 )
 
+func approvalForDeploy(ctx context.Context, b *bundle.Bundle) (bool, error) {
```

Comment:
How does it affect deploying already existing DLT pipelines? I might suspect the change can be a recreate one and thus the behaviour will change, but we need to confirm it

Reply:
Good catch. This PR was meant to only show the prompt for the UC schema resource. I'll address DLT and other resources in a separate PR. @pietern wanted a rundown of the impact this will have since this would be a breaking change post GA.

Reply:
We can't really merge (and release) this change until we know the impact, right?

Reply:
nvm, I see you added a check for schema resources only

Reply:
Since it's only for UC schemas, could you rename the function name so the intention is clear from the name